### DISCUSSION TITLE
Control the "Last Updated" and "Contributors" field location

### DISCUSSION THREAD

**Discussion by rajeshkayarat** [vuepress-theme-hope/vuepress-theme-hope#4069] [Q&A] [answered]
Hi ,

In the default hope theme,is it possible to position the "Last Updated" and  "Contributors" in the header or on the top, instead of having it in the footer.
Currently it comes up in the footer ,is there any possibility of placing it on the top of the page?

**Comment by Mister-Hope** [comment, marked as answer]
https://theme-hope.vuejs.press/guide/advanced/replace.html
